packages/components/src/message/docs/Message.stories.mdx

```diff
@@ -21,6 +21,31 @@ import { TextLink } from "@components/link";
     githubPath="/packages/components/src/message/src"
 />
 
+## Guidelines
+
+### When to use
+
+- To provide feedback such as success, warning, error etc.
+- To confirm an action.
+- To incite the user to take an action, if this action does not require to interrupt the user flow.
+
+### Content
+
+Messages should:
+
+- Be short. Limit your content to one sentence when possible.
+- Provide a path of action. Don't let the user wonder what's next.
+- Use copy that is easily scannable.
+- Link to the appropriate documentation when additional context is needed.
+
+### Accessibility
+
+- Text should be descriptive and complete enough to be understood without relying on the icon. Using an icon to carry information will make it non-accessible.
+
+### Message vs Modal vs Alert
+
+- If the user flow has to be interrupted, use either a [modal](?path=/docs/modal--default-story) or an [alert](?path=/docs/alert--default-story).
```

Comment:
I think it should only be Message vs Alert.

Modal is something else.

Reply:
I agree Modal is too far from a Message.